configserver/src/main/java/com/yahoo/vespa/config/server/deploy/ModelContextImpl.java

```diff
@@ -221,10 +215,10 @@ public String athenzDnsSuffix() {
         public boolean useDedicatedNodeForLogserver() { return useDedicatedNodeForLogserver; }
 
         @Override
-        public boolean useFdispatchByDefault() { return useFdispatchByDefault; }
+        public boolean useFdispatchByDefault() { return false; }
```

Comment:
Remove implementations of `useFdispatchByDefault`/`dispatchWithProtobuf`.

Reply:
Done